gym/envs/phys2d/lunar_lander.py

```diff
@@ -0,0 +1,730 @@
+import warnings
```

Comment:
Basically all of the same comments for lunar lander, remove heuristics and `__main__`, move protobuf and change to jax.numpy